### PR TITLE
Migration/multimeasurerest

### PR DESCRIPTION
**Pull request by rvilarl**
MultiMeasureRest

### CODE REVIEW

File: src/multimeasurerest.ts [hunk 1]
```diff
@@ -3,15 +3,40 @@
...
+  symbol_spacing?: number;
+  serif_thickness: number;
+  use_symbols: boolean;
+  number_line: number;
+  spacing_between_lines_px: number;
+  semibrave_rest_glyph_scale: number;
+  padding_right?: number;
+}
+
+export interface MultimeasureRestSemibraveRest {
```
Comment (0xfe) [inline src/multimeasurerest.ts:32]: This can be inline.

**Review by 0xfe — COMMENTED** [review]
One minor comment

File: src/multimeasurerest.ts (hunk 1 above)
Comment (rvilarl) [reply src/multimeasurerest.ts:32]: done

**Comment by rvilarl** [comment]
ready!

**Comment by rvilarl** [comment]
Rebased, ready to merge!

**Comment by 0xfe** [comment]
LGTM

**Pull request merged**